packages/web/src/common/store/tipping/sagas.ts

```diff
@@ -264,11 +264,19 @@ function* confirmTipIndexed({
 
 function* wormholeAudioIfNecessary({ amount }: { amount: number }) {
   const walletClient = yield* getContext('walletClient')
+  const sdk = yield* getSDK()
+  const { currentUser } = yield* select(getWalletAddresses)
+  if (!currentUser) {
+    throw new Error('Failed to retrieve current user wallet address')
+  }
 
-  const waudioBalanceWei = yield* call([
-    walletClient,
-    'getCurrentWAudioBalance'
-  ])
+  const waudioBalanceWei = yield* call(
+    [walletClient, 'getCurrentWAudioBalance'],
```

Comment:
nit: would be great to conver this to `call([walletClient, walletClient.getCurrentWAudioBalance], ...)`